prow/plugins/trigger/ic_test.go

```diff
@@ -174,14 +282,14 @@ func TestHandleIssueComment(t *testing.T) {
 					Name:      "job",
 					AlwaysRun: true,
 					Context:   "pull-job",
-					Trigger:   "@k8s-bot test this",
+					Trigger:   "(^@k8s-bot test this\r?$|^/test\r?$)",
```

Comment:
Is it necessary for these regex matches to be so restrictive that even white space at end of line will cause the trigger to not fire? It's easy for white space to end up on a line (especially on mobile), and current functionality also allows for putting a comment like "k8s-bot something test this, flake ref:#999999"